internal/audit/hub/conf.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2021-2024 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package hub
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/cerbos/cerbos/internal/audit"
+	"github.com/cerbos/cerbos/internal/audit/local"
+	"github.com/cerbos/cerbos/internal/hub"
+	"go.uber.org/multierr"
+)
+
+const (
+	confKey = audit.ConfKey + ".hub"
+
+	defaultMaxBatchSize     = 16
+	defaultMinFlushInterval = 5 * time.Second
+	defaultFlushTimeout     = 5 * time.Second
+	defaultNumGoRoutines    = 8
+
+	minMinFlushInterval = 2 * time.Second
+	maxFlushTimeout     = 10 * time.Second
+)
+
+var (
+	errInvalidFlushInterval = fmt.Errorf("flushInterval must be at least %s", minMinFlushInterval)
+	errInvalidFlushTimeout  = fmt.Errorf("flushTimeout cannot be more than %s", maxFlushTimeout)
+)
+
+type Conf struct {
+	Ingest     IngestConf `yaml:"ingest"`
```

Comment:
Shall we make this section hidden by adding `conf:",ignore"`? Pretty much all of the settings in it are not things that end users should adjust.

Reply:
If they're going to be hidden, should we just remove them from the config entirely and hardcode them?

Reply:
I think it might be useful to have the escape hatch to change them for testing and debugging purposes.